services/QuillLMS/client/app/bundles/PremiumHub/components/concept_reports_table.jsx

```diff
@@ -5,6 +5,7 @@ import { ReactTable, } from '../../Shared';
 import EmptyStateForReport from '../../Teacher/components/progress_reports/empty_state_for_report';
 
 const ConceptReportsTable = ({ data, }) => {
+  console.log("🚀 ~ file: concept_reports_table.jsx:8 ~ ConceptReportsTable ~ data:", data)
```

Comment:
Do we want this `console.log` call to stay in?

Reply:
@anathomical no, I missed that-- thanks for catching!